learning_resources.md

```diff
@@ -101,6 +101,10 @@
   Introduction to JavaScript and web development.
 * [You Don't Know JavaScript by Kyle Simpson](https://github.com/getify/You-Dont-Know-JS)
   In-depth free eBook on JavaScript.
+  
+### Node
+* [Learn You Node](https://github.com/workshopper/learnyounode)
```

Comment:
Add two space characters at the end of this line (I know, I know...) so a line break appears.